Makefile

```diff
@@ -243,22 +243,67 @@ verify-crd:
 	$(diff) '$(tmp_file)' '$(CRD_PATH)' || (echo 'File $(CRD_PATH) is not up-to date. Please run `make update-crd` to update it.' && false)
 .PHONY: verify-crd
 
-# $1 - target dir
-define generate-dev-manifests
-	mkdir -p '$(1)' && \
-	cp deploy/manager/prod/*.yaml '$(1)' && \
-	yq eval '.spec.cpuset=false | .spec.datacenter.racks[] = (.spec.datacenter.racks[] | .resources.limits.cpu="200m" | .resources.limits.memory="200Mi" | .resources.requests.cpu="10m" | .resources.requests.memory="100Mi" )' -i '$(1)'/10_scylla_scyllacluster.yaml
+# $1 - name
+define generate-namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $(1)
 endef
 
-update-deploy-dev:
-	$(call generate-dev-manifests,deploy/manager/dev)
-.PHONY: update-deploy-dev
+# $1 - name
+# $2 - chart path
+# $3 - values path
+# $4 - target dir
+define generate-manifests
+	mkdir -p '$(4)'
+	$(file > $(4)/00_namespace.yaml,$(generate-namespace))
+	helm template '$(1)' '$(2)' --namespace '$(1)' --values '$(3)' | ./hack/split-helm.sh '$(4)'
+endef
+
+update-deploy-manager-prod:
```

Comment:
nit: Although your version is better for parallelism, I'd be fine if we merge those into just `update-deploy` for simplicity. I am fine with whichever you prefer. Just wanted to write it down so you know you have the option.